### PR TITLE
Helm chart

### PR DESCRIPTION
**Pull request by samiyaakhtar**
Dashboard is running at http://13.89.237.230/? that was deployed using this helm chart, and the command below:

`helm install . --name spektate --set storageAccessKey=*** --set storageTableName=deployments --set storagePartitionKey=hello-bedrock-azure-private --set storageAccountName=saakhtacjteststorage --set pipelineProject=hellobedrockprivate --set pipelineOrg=epicstuff --set pipelineAccessToken=*** --set manifest=hello-bedrock-azure-private-manifest --set manifestAccessToken=*** --set githubManifestUsername=  --set sourceRepoAccessToken=***`

Closes https://github.com/microsoft/bedrock/issues/967

### CODE REVIEW

**Comment by andrebriggs** [comment]
@samiyaakhtar I created https://github.com/microsoft/bedrock/issues/1156 based off of the deployment at http://13.89.237.230/ you linked

File: chart/templates/loadbalancer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: loadbalancer
```
Comment (andrebriggs) [inline chart/templates/loadbalancer.yaml]: With a LoadBalancer type we will expose an external port. Is this want we want to do by default? I think by default we have an internal ip and ask people to port forward in the instructions.

Alternatively we could have a conditional in the `values.yaml` of the Helm Chart to enable a loadbalancer. That think that would be the most flexible thing. Thoughts? @samiyaakhtar @dennisseah

Comment (samiyaakhtar) [reply chart/templates/loadbalancer.yaml]: That's a good idea @andrebriggs 

I can make this change. I'll also add some documentation on how to install via this helm chart

File: chart/values.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+pipelineOrg:
+pipelineAccessToken:
+manifest:
+manifestAccessToken:
+githubManifestUsername:
+sourceRepoAccessToken:
+
+replicaCount: 1
+image:
+  repository: samiyaakhtar/spektate
```
Comment (dennisseah) [inline chart/values.yaml]: `samiyaakhtar/spektate` ?

File: README.md [hunk 3]
```diff
@@ -19,7 +19,30 @@ Note: Spektate dashboard will delete deployments when their corresponding builds
...
+- `storageTableName`: Table name for the storage account
+- `storagePartitionKey`: Partition key for your configuration, you may want to use project name or some identifier that helps separate unrelated configurations for the purpose of introspection.
+- `storageAccountName`: Storage account name
+- `pipelineProject`: Project name for the pipelines in Azure DevOps
+- `pipelineOrg`: Org name for the pipelines in Azure DevOps
+- `pipelineAccessToken`: Access token for pipelines in Azure DevOps
+- `manifestRepoName`: Manifest repository name
+- `manifestAccessToken`: Access token for the manifest repository
+- `sourceRepoAccessToken`: Access token for the source repository
+- **Note**: If you're using GitHub, add `githubManifestUsername`: Account name under which the manifest repository resides.
```
Comment (andrebriggs) [inline README.md]: @samiyaakhtar do you mean the the org name in a GitHub repo? For instance microsoft is the org name in github.com/microsoft/bedrock. If so we should change the variable name

File: chart/templates/loadbalancer.yaml [hunk 4]
```diff
@@ -1,3 +1,4 @@
+{{- if .Values.externalIP -}}
```
Comment (andrebriggs) [inline chart/templates/loadbalancer.yaml]: @samiyaakhtar We might want to add directions on how to port forward if not using a public ip.

**Pull request merged**